### ISSUE TITLE
Error when installing .NET 6 when installing Bicep extension

### ISSUE THREAD

**Issue by intrado-bfhunt** [dotnet/vscode-dotnet-runtime#579]
**Repro steps:**

1. Install Bicep extension

**Extension Version:** 1.0.0
**Error Message:** .NET Acquisition Failed: Installation failed: Error: Command failed: "/home/anonymous/.vscode-server/extensions/ms-dotnettools.vscode-dotnet-runtime-1.5.0/dist/install scripts/dotnet-install.sh" -InstallDir "/home/anonymous/.vscode-server/data/User/globalStorage/ms-dotnettools.vscode-dotnet-runtime/.dotnet/6.0.1" -Version 6.0.1 -Runtime dotnet
/home/anonymous/.vscode-server/extensions/ms-dotnettools.vscode-dotnet-runtime-1.5.0/dist/install scripts/dotnet-install.sh: 7: Syntax error: newline unexpected

**Error Stack:** Error: .NET Acquisition Failed: Installation failed: Error: Command failed: "/home/anonymous/.vscode-server/extensions/ms-dotnettools.vscode-dotnet-runtime-1.5.0/dist/install scripts/dotnet-install.sh" -InstallDir "/home/anonymous/.vscode-server/data/User/globalStorage/ms-dotnettools.vscode-dotnet-runtime/.dotnet/6.0.1" -Version 6.0.1 -Runtime dotnet
/home/anonymous/.vscode-server/extensions/ms-dotnettools.vscode-dotnet-runtime-1.5.0/dist/install scripts/dotnet-install.sh: 7: Syntax error: newline unexpected

	at /home/anonymous/.vscode-server/extensions/ms-dotnettools.vscode-dotnet-runtime-1.5.0/dist/extension.js:2:13750

> Please attach the log file located at /home/bhunt/.vscode-server/data/logs/20220902T160547/exthost1/ms-dotnettools.vscode-dotnet-runtime/DotNetAcquisition-ms-dotnettools.vscode-dotnet-runtime-1662156471081.txt. Note that this file may contain personal data.

The file referenced above was 0 bytes in length.

**Comment by intrado-bfhunt** [comment]
The above exception was when I was referencing a Bicep file inside my WSL2 setup.  I then tried moving my Bicep file onto the Windows side and installing the Bicep extension there and got weirdly similar failures, but in a dotnet-install.ps1 file.  Turns out that `.ps1` was actually a Bash script internally so renaming it to a `.sh` extension allowed it to install .NET 6 correctly and the Bicep extension is working now on the Windows side.

Just flipped back into my WSL2 setup and it looks like the dotnet-install.sh is actually a PoSH script, so it appears the scripts got flipped for extensions.
